src/axom/primal/operators/intersect.hpp

```diff
@@ -1000,6 +1000,7 @@ bool intersect(const Ray<T, 3>& ray,
                axom::Array<T>& v,
                double tol = 1e-8,
                double EPS = 1e-8,
+               bool isTrimmed = true,
```

Comment:
How does this parameter relate to `NURBSPatch::isTrimmed()`?

Reply:
There are some cases where a patch is trimmed, but you want to find intersections with the untrimmed patch, like recording "near-misses" with the visible portion. I should rename this variable to make that more clear, maybe to "treatAsTrimmed" or reverse the implementation and have it be "treatAsUntrimmed"/`considerUntrimmed`?